cloudbuild.yaml

```diff
@@ -20,7 +16,7 @@ steps:
   - id: frontend-cover
     name: 'gcr.io/cloud-builders/npm:current'
     entrypoint: 'make'
-    args: ['-C', 'frontend', 'ensure-coverage']
+    args: ['-C', 'frontend', 'check-coverage']
```

Comment:
these changes to cloudbuild.yaml should be to add all the front-end steps, since until this commit there is no frontend Makefile so cloudbuild is broken.